app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,11 @@
+class CustomersController < ApplicationController
+  def show
+    @customer = Customer.find(params[:customer_id])
+  end
+
+  def update
```

Comment:
So we would not want our adding an `item` to a `customer` to live in our `customer` controller `update` action. The reason being is that if we needed to `update` a customer, we no longer have that CRUD action available to us. 

So what we would want to do is create a `CustomerItems` controller with a `create` action as a `POST`.